contrib/devtools/clang-format.py

```diff
@@ -43,11 +42,11 @@ def check_command_line_args(argv):
 
     if(len(argv) < len(required_args) + 1):
         for word in (['Usage:', argv[0]] + required_args):
-            print word,
-        print ''
+            print(word),
```

Comment:
This got messed up a little. The correct change here is to set `end=''` as an argument to the upper print as the comma does nothing in `python3`. Same for the code below.
You can see it when printing the command line arguments. They are in a row now with this code.

Reply:
Thanks fixed.